.github/workflows/lint.yml

```diff
@@ -0,0 +1,43 @@
+name: golangci
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+permissions:
+  contents: read
+  # Optional: allow read access to pull request. Use with `only-new-issues` option.
+  # pull-requests: read
+jobs:
+  lint:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
+      - uses: actions/checkout@v3
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v3
+        with:
+          # Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
+          version: v1.50.0
+
+          # Optional: working directory, useful for monorepos
+          # working-directory: somedir
+
+          # Optional: golangci-lint command line arguments.
+          args: --skip-dirs=internal/muxado
```

Comment:
I think we should remove this from `lint.yml` and put it in a `.golangci.yml` in the root with:

```yaml

run:
  skip-dirs:
  - internal/muxado
```

The intent is of course that a naive runner of just `golangci-lint` doesn't get warnings or such.